cmake/lint.cmake

```diff
@@ -0,0 +1,35 @@
+if(PROGRAM)
+
+  if(${TARGET} STREQUAL "lintuncrustify")
```

Comment:
why isn't this part of `lintc` ? we don't need separate build targets depending on the tool. We need "lint" tasks, not "lint with a specific tool". The tool is an implementation detail.